example/change-style.md

```diff
@@ -19,3 +20,5 @@ new d3plus.Timeline()
   })
   .render();
 ```
+
+*If you want to change the style of shapes, you must to set `.buttonBehavior("ticks")` for correct behavior in Timeline. Currently "buttons" method doesn't support with `shapeConfig`.
```

Comment:
This is incorrect, buttons does support `shapeConfig` but just in a different way. Remove this disclaimer.